static/app/views/performance/newTraceDetails/traceRenderers/virtualizedViewManager.tsx

```diff
@@ -1177,97 +1191,23 @@ export class VirtualizedViewManager {
     return [0, right_outside];
   }
 
+  last_indicator_width = 0;
   draw(options: {list?: number; span_list?: number} = {}) {
     const list_width = options.list ?? this.columns.list.width;
     const span_list_width = options.span_list ?? this.columns.span_list.width;
 
-    if (this.divider) {
-      this.divider.style.setProperty(
-        '--translate-x',
-        // @ts-expect-error we set number value type on purpose
-        Math.round(
-          (list_width * (this.container_physical_space.width - this.scrollbar_width) -
-            DIVIDER_WIDTH / 2 -
-            1) *
-            10
-        ) / 10
-      );
-    }
-    if (this.indicator_container) {
-      const correction =
-        (this.scrollbar_width / this.container_physical_space.width) * span_list_width;
-      // @ts-expect-error we set number value type on purpose
-      this.indicator_container.style.setProperty('--translate-x', -this.scrollbar_width);
-      this.indicator_container.style.width = (span_list_width - correction) * 100 + '%';
-    }
-
-    if (this.container) {
-      this.container.style.setProperty(
-        '--list-column-width',
-        // @ts-expect-error we set number value type on purpose
-        Math.round(list_width * 1000) / 1000
-      );
-      this.container.style.setProperty(
-        '--span-column-width',
-        // @ts-expect-error we set number value type on purpose
-        Math.round(span_list_width * 1000) / 1000
-      );
-    }
+    this.drawContainers(this.container, {
+      list_width,
+      span_list_width,
+    });
 
     for (let i = 0; i < this.columns.list.column_refs.length; i++) {
-      const span_bar = this.span_bars[i];
```

Comment:
All of these methods got consolidated into the appropriate drawXXX calls, hence why the diff got huge. It was unmaintainable to keep them all inline